test/stablex/batch_exchange.js

```diff
@@ -39,9 +39,7 @@ const smallTradeData = {
 }
 
 contract("BatchExchange", async accounts => {
-  const solver = accounts.pop()
-  const competingSolver = accounts.pop()
-  const [user_1, user_2, user_3] = accounts
+  const [user_1, user_2, user_3, solver, competingSolver] = accounts
```

Comment:
There was a reason we had done this in the past. This was introduced when @nlordell  and I setup the Example files. Perhaps it is no longer necessary.